lib/endpoints/class-wp-rest-users-controller.php

```diff
@@ -517,6 +517,17 @@ public function prepare_item_for_response( $user, $request ) {
 		}
 
 		$context = ! empty( $request['context'] ) ? $request['context'] : 'embed';
+
+		if ( 'edit' === $context ) {
```

Comment:
Let's check like this:

```
if ( isset( $data['capabilities'] ) && empty( $data['capabilities'] ) ) {
    $data['capabilities'] = new stdClass();
}
```

This way, we'll always fix our data regardless of `context`.